apps/definition_dictionary/lib/dictionary/type/string.ex

```diff
@@ -2,6 +2,12 @@ defmodule Dictionary.Type.String do
   use Definition, schema: Dictionary.Type.String.V1
   use Dictionary.JsonEncoder
 
+  @type t :: %__MODULE__{
+    version: integer,
+    name: String.t(),
+    description: String.t()
+  }
```

Comment:
many type definitions lost their lives to bring us that big source/destination refactor... 😞